ezgripper_controller:
  type: "position_controllers/JointTrajectoryController"
  joints:
    - left_ezgripper_knuckle_1
    - left_ezgripper_knuckle_2
    - left_ezgripper_knuckle_3
    - left_ezgripper_knuckle_4

gazebo_ros_control:
  pid_gains:
    left_ezgripper_knuckle_1: {p: 100.0, i: 0.01, d: 10.0}
    left_ezgripper_knuckle_2: {p: 100.0, i: 0.01, d: 10.0}
    left_ezgripper_knuckle_3: {p: 100.0, i: 0.01, d: 10.0}
    left_ezgripper_knuckle_4: {p: 100.0, i: 0.01, d: 10.0}